app/main/RTMedia.php

```diff
@@ -1158,13 +1158,20 @@ function enqueue_scripts_styles() {
 		$request_uri = rtm_get_server_var( 'REQUEST_URI', 'FILTER_SANITIZE_URL' );
 		$url          = trailingslashit( $request_uri );
 		$rtmedia_slug = '/' . RTMEDIA_MEDIA_SLUG;
-		// check position of media slug from end of the URL
+		// check position of media slug from end of the URL.
 		if ( strrpos( $url, $rtmedia_slug ) !== false ) {
-			// split the url upto the last occurance of media slug
+			// split the url upto the last occurrence of media slug.
 			$url_upload = substr( $url, 0, strrpos( $url, $rtmedia_slug ) );
 			$url        = trailingslashit( $url_upload ) . 'upload/';
 		} else {
-			$url = trailingslashit( $url ) . 'upload/';
+
+			// If url contains '?' then put query string at last.
+			if ( strstr( $url, '?' ) ) {
+				$url = explode( '?', $url );
+				$url = trailingslashit( $url[0] ) . 'upload/?' . trim( $url[1], '/' );
```

Comment:
@BhargavBhandari90 Please add sanity check before using Exploded variables $url[0] and $url[1].